test/cpp_extensions/open_registration_extension/torch_openreg/cmake/TorchPythonTargets.cmake

```diff
@@ -0,0 +1,36 @@
+if(WIN32)
+  set(TORCH_PYTHON_IMPORTED_LOCATION "${PYTORCH_INSTALL_DIR}/lib/torch_python.lib")
+  set(TORCH_PYTHON_LINK_LIBS
+    "${PYTORCH_INSTALL_DIR}/lib/c10.lib"
+    "${PYTORCH_INSTALL_DIR}/lib/torch_cpu.lib"
+  )
+  set(TORCH_PYTHON_LIBRARY_LINK
+    "\$<TARGET_FILE:torch_python>;\$<TARGET_PROPERTY:torch_python,INTERFACE_LINK_LIBRARIES>"
+  )
+else()
+  set(TORCH_PYTHON_IMPORTED_LOCATION "${PYTORCH_INSTALL_DIR}/lib/libtorch_python.so")
+  set(TORCH_PYTHON_LINK_LIBS "c10;torch_cpu")
+  set(TORCH_PYTHON_LIBRARY_LINK
+    "-Wl,--no-as-needed,\"\$<TARGET_FILE:torch_python>\" -Wl,--as-needed;"
+    "\$<TARGET_PROPERTY:torch_python,INTERFACE_LINK_LIBRARIES>"
+  )
+endif()
```

Comment:
Why are all of these set by hand?
Also shouldn't we get these from the torch_python library below?